python/src/worker/tasks/cluster_cells.py

```diff
@@ -1,12 +1,9 @@
 import json
-from logging import info
 
 import backoff
-import pandas as pd
 import requests
 from aws_xray_sdk.core import xray_recorder
 from exceptions import raise_if_error
-from natsort import natsorted
```

Comment:
you can also delete natsort from requirements -- it's not used in the worker anymore

Reply:
done